rosidl_typesupport_opensplice_c/resource/msg__type_support_c.cpp.em

```diff
@@ -61,11 +61,6 @@ extern "C"
 {
 #endif
 
-// Forward declare the get type support function for this type.
-ROSIDL_GENERATOR_C_EXPORT_@(spec.base_type.pkg_name)
-const rosidl_message_type_support_t *
-  ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))();
-
```

Comment:
Yeah... I looked into it a bit. My original idea that the `type_support.h` file gets included by one of the other headers looks like it's not correct. It might work because there is still a forward declaration for ROSIDL_GET_TYPE_SUPPORT_FUNCTION for message subfields lower in the file: